docs/variants/workstation/installation-manual.md

```diff
@@ -0,0 +1,218 @@
+
+    ![](../../images/service_mode_warn.jpg)
+
+## Installation
+
+You will need root privileges from now on to proceed. So switch to root user or
+use __*sudo*__ in each command.
+
+1. Install flashrom v1.1 with your distribution's package manager, if you don't
+   have it installed yet.
+2. Back your firmware image up with: `sudo flashrom -p internal -r bios_backup.bin`
+   (be sure flashrom doesn't report any errors like below).
+3. Download the [Dasharo Workstation firmware image](https://cloud.3mdeb.com/index.php/s/8WNEHEFcBGFRK23)
+4. Flash it on you Dell OptiPlex machine:
+
+   ``` console
+   sudo flashrom -p internal --ifd -i bios -i me -w <path_to_the_binary_file>
+   ```
+   
+   for example: 
+
+   ``` console
+   $ sudo flashrom -p internal --ifd -i bios -i me -w /tmp/dasharo_workstation_v0.2_rc3.rom 
+      
+   flashrom v1.1-rc1-127-g370a9f3 on Linux 4.19.0-9-amd64 (x86_64)
+   flashrom is free software, get the source code at https://flashrom.org
+
+   Using clock_gettime for delay loops (clk_id: 1, resolution: 1ns).
+   Found chipset "Intel Q77".
+   This chipset is marked as untested. If you are using an up-to-date version
+   of flashrom *and* were (not) able to successfully update your firmware with
+   it, then please email a report to flashrom@flashrom.org including a verbose
+   (-V) log.
+   Thank you!
+   Enabling flash write... SPI Configuration is locked down.
+   The Flash Descriptor Override Strap-Pin is set. Restrictions implied by
+   the Master Section of the flash descriptor are NOT in effect. Please note
+   that Protected Range (PR) restrictions still apply.
+   Enabling hardware sequencing due to multiple flash chips detected.
+   OK.
+   Found Programmer flash chip "Opaque flash chip" (12288 kB,
+      Programmer-specific) mapped at physical address 0x0000000000000000.
+   Reading old flash chip contents... done.
+   Erasing and writing flash chip... Erase/write done.
+   Verifying flash... VERIFIED.
+   ```
+
+   If you get a warning:
+
+   ``` console
+   WARNING! You may be running flashrom on an unsupported laptop.
+   ```
+
+   And programmer initialization failed, run command:
+
+   ``` console
+   flashrom -p internal:laptop=this_is_not_a_laptop -w /tmp/dasharo_workstation_v0.2_rc3.rom --ifd -i bios -i me
+   ```
+
+   If you have placed the jumper correctly you should see the following message
+   in flashrom's output:
+
+   ``` console
+   The Flash Descriptor Override Strap-Pin is set. Restrictions implied by
+   the Master Section of the flash descriptor are NOT in effect. Please note
+   that Protected Range (PR) restrictions still apply.
+   ```
+
+   Newer version of flashrom may not display the warning about unsupported
+   chipset as it already may be marked as tested. Our team has verified that the
+   flashrom updates firmware reliably on this chipset.
+
+## Verification
+
+1. If everything went well (flashrom has verified the flash content),
```

Comment:
1. ... :
2. Shut down **the** machine, move **a** jumper to the original place

Reply:
@mikebdp2 grammaly applied